src/lib/gitlab.ts

```diff
@@ -20,6 +20,35 @@ export class Gitlab {
         return hide;
     }
 
+     private static makeCommitData(commit: any, branchName: string): ICommitData {
```

Comment:
You forgot to specify `avatar` here
`private static makeCommitData(commit: any, avatar: string, branchName: string)`